koku/api/test_query_params.py

```diff
@@ -684,8 +684,8 @@ def test_process_tag_query_params(self):
             "filter[resolution]=monthly&"
             "filter[time_scope_value]=-1&"
             "filter[time_scope_units]=month&"
-            "filter[tag:environment]=prod&"
-            "group_by[tag:app]=*"
+            "filter%5Btag:environment]=prod&"  # urlencoded search now use for tag
+            "group_by%5Btag:app]=stage"
```

Comment:
Should we have test scenarios for both URL encoded and text values?

Reply:
I still think it's worth testing URL encoded values being sent to `QueryParameters._validate()`. Maybe that's not what this was testing.